### PR TITLE
Minutes asc269

### PR DESCRIPTION
**Pull request by andrewschoi**
When visiting the Employees page on Admin, users can now see driver's availability with minutes

### CODE REVIEW

File: frontend/src/components/EmployeeModal/EmployeeModal.tsx [hunk 1]
```diff
@@ -53,12 +53,13 @@ const EmployeeModal = ({ existingEmployee }: EmployeeModalProps) => {
   );
   const [imageBase64, setImageBase64] = useState('');
   const { withDefaults } = useReq();
-  const { refreshAdmins, refreshDrivers } = useEmployees();
+  const { refreshAdmins, refreshDrivers, drivers } = useEmployees();
```
Comment (tuckerstanley) [inline frontend/src/components/EmployeeModal/EmployeeModal.tsx:56]: What do you use drivers for? It doesnt seem like it is used in employeeModal at all.

File: frontend/src/components/UserDetail/EmployeeDetail.tsx [hunk 2]
```diff
@@ -127,6 +127,7 @@ const findEmployee = (
   const employee = DriverToEmployees(drivers).find(
     (employee) => employee.id === employeeId
   );
+  console.log(drivers);
```
Comment (tuckerstanley) [inline frontend/src/components/UserDetail/EmployeeDetail.tsx:130]: Delete! We don't want console.logs getting merged in, they should just be used for testing purposes.

**Review by tuckerstanley — CHANGES_REQUESTED** [review]
Looking good! See my comments, but these should be quick fixes! Nice job.

**Review by epicdragon44 — APPROVED** [review]
Looks like all of Tucker's issues were addressed. Approving for merge.

**Pull request merged**